lib/ansible/module_utils/gitlab.py

```diff
@@ -3,6 +3,7 @@
 # (c) 2018, Marcus Watkins <marwatk@marcuswatkins.net>
```

Comment:
Format of copyright statements is now different.
```suggestion
# Copyright: (c) 2018, Marcus Watkins <marwatk@marcuswatkins.net>
```

Reply:
Will be fix in the next commit